languages/python/oso/polar/polar.py

```diff
@@ -84,6 +84,9 @@ def __init__(self, classes=CLASSES):
         self.register_class(str, name="String")
         self.register_class(datetime, name="Datetime")
         self.register_class(timedelta, name="Timedelta")
+        self.register_class(OsoActor, name="OsoActor", entity_type="OsoActor")
```

Comment:
This probably isn't a big deal, but it's a little weird that a user could start doing things like `actor = new OsoActor()` and it's going to create a random instance. Not sure there's any need to fix this, but there might be similar implications?